src/redux/catagories/catagoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const catagoriesSlice = createSlice({
+  name: 'catagory',
+  initialState,
+  reducers: {
+    checkStatus: (state) => state,
```

Comment:
- [ ] The category reducer should always return the text "Under Construction". Kindly refactor to that. See this example:

  ```
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction" ? "Under construction" : state.categories
    },
  ```